rocAL/rocAL_pybind/example/coco_pipeline.py

```diff
@@ -162,12 +192,15 @@ def __next__(self):
         self.lis = []  # Empty list for bboxes
         self.lis_lab = []  # Empty list of labels
 
-        if(types.NCHW == self.tensor_format):
-            self.loader.copyToTensorNCHW(
-                self.out, self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
+        if (self.device == "cpu"):
+            if(types.NCHW == self.tensor_format):
+                self.loader.copyToTensorNCHW(
+                    self.out, self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
+            else:
+                self.loader.copyToTensorNHWC(
+                    self.out, self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
         else:
-            self.loader.copyToTensorNHWC(
-                self.out, self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
+            self.get_cupy_array_from_address()
```

Comment:
It is better to call copy fun here and pass the pointer to get_cupy_array..()

Reply:
Made this change